### ISSUE TITLE
Clear redis cache on start up

### ISSUE THREAD

**Issue by Laurin-W** [activitypods/activitypods#262]
I just had to manually remove a user that was only semi-created and that caused a database inconsistency which prevented the server from starting up. I manually deleted the user from the fuseki database but after the restart I noticed that I still need to clear the redis cache for moleculer actions since that is kept across sessions.

I think it would make sense to delete the cache on start up to make it easier to do maintenance work.

**Comment by srosset81** [comment]
I agree. There is a slight performance loss after cache clearing, but it is not comparable to the annoyance of having outdated cache.
